orchagent/copporch.cpp

```diff
@@ -0,0 +1,540 @@
+#include "sai.h"
+
+#include "copporch.h"
+#include "logger.h"
+
+#include <sstream>
+#include <iostream>
+
+extern sai_hostif_api_t*    sai_hostif_api;
+extern sai_policer_api_t*   sai_policer_api;
+
+CoppOrch::CoppOrch(DBConnector *db, string tableName) :
+    Orch(db, tableName)
+{
+    SWSS_LOG_ENTER();
+};
+
+bool CoppOrch::getPolicerMeter(string input, sai_meter_type_t& meter_value) const
+{
+    SWSS_LOG_ENTER();
+    SWSS_LOG_DEBUG("copp policer meter type:%s\n", input.c_str());
+    if (input == copp_policer_meter_packets)
+    {
+        meter_value = SAI_METER_TYPE_PACKETS;
+    }
+    else if (input == copp_policer_meter_bytes)
+    {
+        meter_value = SAI_METER_TYPE_BYTES;        
+    }
+    else
+    {
+        SWSS_LOG_DEBUG("invalid meter value :%s\n", input.c_str());
+        return false;
+    }
+    return true;
```

Comment:
use map? std::map<std::string, enum>

you can define a meter map to translate the name to enum.